scripts/lint-staged/src/eslint.js

```diff
@@ -7,7 +7,7 @@ const os = require('os');
 const path = require('path');
 const { promisify } = require('util');
 
-const { getWorkspaceProjects, workspaceRoot } = require('@fluentui/scripts-monorepo');
+const { findConfig } = require('@fluentui/scripts-utils');
```

Comment:
obtaining project is now lazy instead of fetching all projects and trying to match against their root paths. we no longer use `getProjects` from nx DAEMON